<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1352</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="iconSize">
   <size>
    <width>25</width>
    <height>25</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTreeView" name="treeView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>220</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTextEdit" name="textEdit">
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1352</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>笔记</string>
    </property>
    <addaction name="action_Ctrl_N"/>
    <addaction name="action_8"/>
    <addaction name="actSetting"/>
    <addaction name="action"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="actBold"/>
    <addaction name="actItalic"/>
    <addaction name="actUnderline"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>窗口</string>
    </property>
    <addaction name="action_6"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_7"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actBold"/>
   <addaction name="actItalic"/>
   <addaction name="actUnderline"/>
   <addaction name="separator"/>
   <addaction name="actH1"/>
   <addaction name="actH2"/>
   <addaction name="actH3"/>
   <addaction name="actH4"/>
   <addaction name="actH5"/>
   <addaction name="actH6"/>
   <addaction name="separator"/>
   <addaction name="actAlignL"/>
   <addaction name="actAlignC"/>
   <addaction name="actAlignR"/>
   <addaction name="separator"/>
   <addaction name="actAddList"/>
   <addaction name="actRandomList"/>
   <addaction name="separator"/>
   <addaction name="actAddImg"/>
   <addaction name="actTable"/>
  </widget>
  <widget class="QToolBar" name="toolBar_3">
   <property name="windowTitle">
    <string>toolBar_3</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="actLearn"/>
   <addaction name="separator"/>
   <addaction name="actNewNote"/>
   <addaction name="actOutput"/>
   <addaction name="actInput"/>
   <addaction name="actDel"/>
   <addaction name="actMenu"/>
   <addaction name="separator"/>
   <addaction name="actLast"/>
   <addaction name="actNext"/>
   <addaction name="separator"/>
   <addaction name="actClip"/>
   <addaction name="actCloze"/>
   <addaction name="separator"/>
   <addaction name="actFind"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Ctrl_N">
   <property name="text">
    <string>新建笔记(Ctrl+N)</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actSetting">
   <property name="text">
    <string>设置</string>
   </property>
  </action>
  <action name="actBold">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/img/res/icon/bold.png</normaloff>:/img/res/icon/bold.png</iconset>
   </property>
   <property name="text">
    <string>加粗</string>
   </property>
  </action>
  <action name="actItalic">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/img/res/icon/italic.png</normaloff>:/img/res/icon/italic.png</iconset>
   </property>
   <property name="text">
    <string>斜体</string>
   </property>
  </action>
  <action name="actUnderline">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/img/res/icon/underLine.png</normaloff>:/img/res/icon/underLine.png</iconset>
   </property>
   <property name="text">
    <string>下划线</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>布局</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>打开文件夹</string>
   </property>
  </action>
  <action name="actH1">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/img/res/icon/h1.png</normaloff>:/img/res/icon/h1.png</iconset>
   </property>
   <property name="text">
    <string>标题1</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actH2">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/img/res/icon/h2.png</normaloff>:/img/res/icon/h2.png</iconset>
   </property>
   <property name="text">
    <string>标题2</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actAlignL">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/img/res/icon/alignLeft.png</normaloff>:/img/res/icon/alignLeft.png</iconset>
   </property>
   <property name="text">
    <string>左对齐</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actAlignR">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/img/res/icon/alignRight.png</normaloff>:/img/res/icon/alignRight.png</iconset>
   </property>
   <property name="text">
    <string>右对齐</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actAlignC">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/img/res/icon/alignCenter.png</normaloff>:/img/res/icon/alignCenter.png</iconset>
   </property>
   <property name="text">
    <string>居中对齐</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actAddList">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/img/res/icon/list.png</normaloff>:/img/res/icon/list.png</iconset>
   </property>
   <property name="text">
    <string>添加列表</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actRandomList">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/img/res/icon/randomList.png</normaloff>:/img/res/icon/randomList.png</iconset>
   </property>
   <property name="text">
    <string>无序列表</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actH3">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/img/res/icon/h3.png</normaloff>:/img/res/icon/h3.png</iconset>
   </property>
   <property name="text">
    <string>标题3</string>
   </property>
   <property name="toolTip">
    <string>标题3</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actH4">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/img/res/icon/h4.png</normaloff>:/img/res/icon/h4.png</iconset>
   </property>
   <property name="text">
    <string>标题4</string>
   </property>
   <property name="toolTip">
    <string>标题4</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actH5">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/img/res/icon/h5.png</normaloff>:/img/res/icon/h5.png</iconset>
   </property>
   <property name="text">
    <string>标题5</string>
   </property>
   <property name="toolTip">
    <string>标题5</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actH6">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/img/res/icon/h6.png</normaloff>:/img/res/icon/h6.png</iconset>
   </property>
   <property name="text">
    <string>正文</string>
   </property>
   <property name="toolTip">
    <string>标题6</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actAddImg">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/img/res/icon/iamge.png</normaloff>:/img/res/icon/iamge.png</iconset>
   </property>
   <property name="text">
    <string>插入图片</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actOutput">
   <property name="text">
    <string>保存</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actInput">
   <property name="text">
    <string>读取</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actLast">
   <property name="text">
    <string>上一个</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actNext">
   <property name="text">
    <string>下一个</string>
   </property>
   <property name="toolTip">
    <string>下一个</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actClip">
   <property name="text">
    <string>剪辑</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actCloze">
   <property name="text">
    <string>挖空</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actMenu">
   <property name="text">
    <string>目录</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actLearn">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/img/res/icon/learn.png</normaloff>:/img/res/icon/learn.png</iconset>
   </property>
   <property name="text">
    <string>学习模式</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actTable">
   <property name="icon">
    <iconset resource="icon.qrc">
     <normaloff>:/img/res/icon/table.png</normaloff>:/img/res/icon/table.png</iconset>
   </property>
   <property name="text">
    <string>插入表格</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actNewNote">
   <property name="text">
    <string>新建</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actFind">
   <property name="text">
    <string>查找</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="actDel">
   <property name="text">
    <string>删除</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
